pandas/core/groupby.py

```diff
@@ -2834,6 +2834,17 @@ def aggregate(self, func_or_funcs, *args, **kwargs):
 
     def _aggregate_multiple_funcs(self, arg, _level):
         if isinstance(arg, dict):
+
+            # show the deprecation, but only if we
+            # have not shown a higher level one
+            # GH 15931
+            if isinstance(self._selected_obj, Series) and _level <= 1:
+                warnings.warn(
+                    ("using a dict on a Series for aggregation\n"
+                     "is deprecated and will be removed in a future "
+                     "version"),
+                    FutureWarning, stacklevel=7)
```

Comment:
the real issue is that the agg evaulation is recursive (somewhat), so I *could* figure it out I suppose but...